tensorboard/backend/application.py

```diff
@@ -85,6 +88,8 @@ def TensorBoardWSGIApp(
       deprecated_multiplexer: Optional `plugin_event_multiplexer.EventMultiplexer`
           to use for any plugins not yet enabled for the DataProvider API.
           Required if the data_provider argument is not passed.
+      auth_providers: Optional mapping whose values are `AuthProvider` values
+          (with arbitrary keys). Defaults to `{}`.
```

Comment:
The keys are arbitrary in what actual object they can be, but not semantically arbitrary, if the expectation is that data providers will reference values from this mapping using those keys - maybe clarify that aspect?

Reply:
Good point; thanks. Will clarify.

Reply:
Done.